src/Integration.UnitTests/State/StateManagerTests.cs

```diff
@@ -425,30 +428,6 @@ public void StateManager_Dispose()
             connection1.IsDisposed.Should().BeTrue("Leaking connections?");
         }
 
-        [TestMethod]
-        public void StateManager_GetConnectedServer()
```

Comment:
I deleted this method from the interface. I'll add explanatory review comments at the appropriate places in the code.